app/controllers/landing_page_controller.rb

```diff
@@ -1,5 +1,12 @@
 class LandingPageController < ApplicationController
   def show
     @form = SearchForm.new
```

Comment:
Just leaving a note in here... Now we've introduced the `RemoteSearchForm` for remote advice, the naming of `SearchForm` is too generic. As we discussed in person - we should rename this to `InPersonSearchForm` to better reflect the direction this is taking, and to differentiate between the different types of advice.